include/wx/uilocale.h

```diff
@@ -138,11 +138,68 @@ class WXDLLIMPEXP_BASE wxUILocale
     // its dtor is not virtual.
     ~wxUILocale();
 
+    // Try to get user's (or OS's) preferred language setting.
+    // Return wxLANGUAGE_UNKNOWN if language-guessing algorithm failed
+    static int GetSystemLanguage();
+
+    // get the encoding used by default for text on this system, returns
+    // wxFONTENCODING_SYSTEM if it couldn't be determined
+    static wxFontEncoding GetSystemEncoding();
```

Comment:
I don't think we need this function any more, font encodings are only useful in non-Unicode builds and we'll be dropping support for them soon, so why bother adding new functions used by them only.

Reply:
Ok, I wasn't aware of that. So, I will revert the change.